src/manage.c

```diff
@@ -323,17 +324,14 @@ static void manage_acq()
     return;
   }
 
-  log_info("acq: PRN %d found @ %d Hz, %d SNR\n", prn + 1, (int)cf, (int)snr);
```

Comment:
Added an issue to piksi_tools to replace this print with a view: https://github.com/swift-nav/piksi_tools/issues/124